src/test/java/com/wikia/webdriver/elements/mercury/components/TopBar.java

```diff
@@ -116,6 +117,20 @@ public boolean isSearchIconVisible() {
     }
   }
 
+  public boolean isSearchIconClickable() {
+    try {
+      searchIconClickableLink.isDisplayed();
+      wait.forElementClickable(searchIconClickableLink, 0);
+      return true;
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)